src/fsharp/FSharp.Core/async.fs

```diff
@@ -305,7 +305,7 @@ namespace Microsoft.FSharp.Control
             contents.aux.ccont (OperationCanceledException (contents.aux.token))
 
         /// Check for trampoline hijacking.
-        // Note, this must make tailcalls, so may not be an instance member taking a byref argument,
+        /// Note, this must make tailcalls, so may not be an instance member taking a byref argument,
```

Comment:
Looks like an internal comment to me. @dsyme Could you advice?

Reply:
@auduchinok, here it is rather suspicious since the line below is a continuation of this one and it begins with `///`

Reply:
Yeah, I thought maybe it was better to change all comments below to `//` here?

Reply:
oh, below in the same file there is a similar comment that uses `//`
![image](https://user-images.githubusercontent.com/26364714/132519348-deb34251-df94-4c0e-85da-6c932ae28673.png)